packages/mui/src/components/breadcrumb/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { useBreadcrumb, useRefineContext } from "@pankod/refine-core";
+import {
+    Breadcrumbs as MuiBreadcrumbs,
+    BreadcrumbsProps as MuiBreadcrumbProps,
+    Typography,
+    Link,
+    LinkProps,
+} from "@mui/material";
+
+import { HomeOutlined } from "@mui/icons-material";
+
+export type BreadcrumbProps = {
+    breadcrumbProps?: MuiBreadcrumbProps;
+    linkProps?: LinkProps;
+    showHome?: boolean;
+    hideIcons?: boolean;
+};
+
+export const Breadcrumb: React.FC<BreadcrumbProps> = ({
+    breadcrumbProps,
+    linkProps,
+    showHome = true,
+    hideIcons = false,
+}) => {
+    const { breadcrumbs } = useBreadcrumb();
+    const { hasDashboard } = useRefineContext();
+
+    if (breadcrumbs.length === 1) {
+        return null;
+    }
+
+    return (
+        <MuiBreadcrumbs
+            {...breadcrumbProps}
+            aria-label="breadcrumb"
+            sx={{
+                paddingY: 2,
+                paddingX: 2,
+                ...(breadcrumbProps?.sx ?? {}),
+            }}
+        >
+            {showHome && hasDashboard && (
+                <Link
+                    underline="hover"
+                    {...linkProps}
+                    sx={{ display: "flex", alignItems: "center" }}
+                    color="inherit"
+                    href="/"
```

Comment:
```suggestion
                    href="/"
                    to="/"
```
For Next.js we have to add "to"